client/src/components/new-request/grey-box.vue

```diff
@@ -140,25 +134,23 @@ import { matchWord, removeExcessSpaces, replaceWord } from '@/plugins/utilities'
   components: { ReserveSubmit }
 })
 export default class GreyBox extends Vue {
+  @Prop(Boolean) changesInBaseName: boolean
+  @Prop(Boolean) designationIsFixed: boolean
   @Prop(Number) i: number
   @Prop(Number) issueIndex: number
   @Prop(Object) option: OptionI
   @Prop(String) originalName: string
 
-  @Watch('changesInBaseName', { immediate: true })
```

Comment:
These were no longer needed now that related getters exist in the parent component.